plugins/copr.py

```diff
@@ -138,17 +139,16 @@ def configure(self):
         config_path = self.base.conf.pluginconfpath[0]
 
         default_config_file = os.path.join(config_path, PLUGIN_CONF + ".conf")
+        # would use appdirs, but that would mean a new dependency
+        vendor_config_file = os.path.join("/usr/share/dnf/plugins/", PLUGIN_CONF + ".vendor.conf")
+
         if os.path.isfile(default_config_file):
             config_files.append(default_config_file)
 
-            copr_plugin_config.read(default_config_file)
-            if copr_plugin_config.has_option('main', 'distribution') and\
-                    copr_plugin_config.has_option('main', 'releasever'):
-                distribution = copr_plugin_config.get('main', 'distribution')
-                releasever = copr_plugin_config.get('main', 'releasever')
-                self.chroot_config = [distribution, releasever]
-            else:
-                self.chroot_config = [False, False]
+        if os.path.isfile(vendor_config_file):
+            self._get_copr_chroot(vendor_config_file)
+        else:
+            self._get_copr_chroot(default_config_file)
```

Comment:
This is wrong. You read vendor_config OR default_config. You should read both. It should be:
```
        if os.path.isfile(vendor_config_file):
            self._get_copr_chroot(vendor_config_file)
        self._get_copr_chroot(default_config_file)
```